colin/core/target.py

```diff
@@ -25,7 +26,7 @@ def _get_target_instance(target_name):
         :param target_name: str
         :return: Container/Image
         """
-        with DockerBackend() as backend:
+        with DockerBackend(logging_level=logging.NOTSET) as backend:
```

Comment:
does this mean "no logs"?

Reply:
I just need to turn off the `conu` logs. Is there any better approach to do this?

Reply:
don't know; if this works, we're good, if not, we may have to fix that in conu (something like: "don't setup logging")